test/sortitionPoolTest.js

```diff
@@ -378,17 +378,38 @@ describe("SortitionPool", () => {
       expect(ineligibleReward).to.equal(200)
     })
 
-    it("sets operator ineligibility correctly", async () => {
+    it("sets and restores operator eligibility correctly", async () => {
       await token.connect(deployer).mint(deployer.address, 1000)
       await pool.connect(owner).insertOperator(alice.address, 10000)
       await pool.connect(owner).insertOperator(bob.address, 20000)
+
       const now = await helpers.time.lastBlockTime()
       const bobID = await pool.getOperatorID(bob.address)
+
       await pool.connect(owner).setRewardIneligibility([bobID], now + 100)
 
+      await expect(await pool.isEligibleForRewards(bob.address)).to.be.false
```

Comment:
All the `await` before `expect` can be removed.

Reply:
🙈 7038dcb3d6d9a592b2f0a19ef78b006088df21ae